File: 프로그래머스/2/12951. JadenCase 문자열 만들기/JadenCase 문자열 만들기.py
def solution(s):

    slist = list(map(str,s.split(" ")))
    answerlist = []

    for i in range(len(slist)) :
        if slist[i] == "":
            answerlist.append("")
            continue
        
        appendstr = slist[i][0].upper()
        appendstr+=slist[i][1:].lower()
        answerlist.append(appendstr)
        
    answer = ' '.join(map(str,answerlist))

    
    return answer

File: 프로그래머스/2/12951. JadenCase 문자열 만들기/JadenCase 문자열 만들기.java
import java.util.*;
class Solution {
    public String solution(String s) {
        String[] words = s.split(" ", -1);  
        StringBuilder answer = new StringBuilder();
        
        for (int i = 0; i < words.length; i++) {
            if (words[i].length() > 0) {
                answer.append(words[i].substring(0, 1).toUpperCase())
                      .append(words[i].substring(1).toLowerCase());
            }
            if (i < words.length - 1) {
                answer.append(" ");
            }
        }
        
        return answer.toString();
    }
}